app/HutchManager/Dockerfile

```diff
@@ -1,9 +1,11 @@
 ﻿FROM node:lts-alpine as spa_build
+COPY pnpm-lock.yaml .
+COPY pnpm-workspace.yaml .
+COPY package.json .
+COPY ./app/manager-frontend /manager-frontend
 RUN npm install -g pnpm
```

Comment:
we should enable corepack, not manually install pnpm. how do we know what version of pnpm it will use?